### PR TITLE
Init a file for Encinitas CA

### LINKED ISSUE #17: Add a hometown file for johnsgill3
## Hometown file
- [x] Create a branch
- [x] Add a file
- [x] Make some commits
- [x] Open a pull request
- [ ] Have a discussion
- [ ] Merge Pull Request

@jfriisTD Should I add anything else?

### PR DESCRIPTION
**Pull request by johnsgill3**
Added a couple of places to eat and fun things to do! Closes #17

### CODE REVIEW

File: places/jsg-encinitas.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+### Places to Eat
+- Rico's Taco Shop
+- Tony's Jacal
+- Native Foods Cafe
+
+### Things to do
+- Go to the beach
+- Go for a bike ride along Hwy 101
+- Eat Mexican Food
```
Comment (johnsgill3) [inline places/jsg-encinitas.md:13]: Where else would you recommend @jfriisTD :question:

Comment (jfriisTD) [inline places/jsg-encinitas.md:13]: paddle!!

Comment (johnsgill3) [inline places/jsg-encinitas.md:13]: That's a great :bulb:, Will make sure to add it!

Comment (jfriisTD) [inline places/jsg-encinitas.md:13]: um. during the break. multi tasking functions overloaded ...

**Pull request merged**